src/main/java/org/sagebionetworks/bridge/dynamodb/DynamoSurveyDao.java

```diff
@@ -85,6 +85,8 @@ QueryBuilder setDeleted(boolean includeDeleted) {
             List<DynamoSurvey> dynamoSurveys = null;
             if (surveyGuid == null) {
                 dynamoSurveys = queryBySecondaryIndex();
+            } else if (surveyGuid.startsWith("identifier:")) { 
```

Comment:
I would use some constants: "identifier:" can be a string constant, and then instead of a magic number (11), you can use the constant string's length, consolidating any future change in one place.

Trivial: in the account case, the prefix is case insensitive, this should be case insensitive too.